airbyte/cloud/connections.py

```diff
@@ -256,6 +256,54 @@ def get_sync_result(
             job_id=job_id,
         )
 
+    def rename(self, name: str) -> CloudConnection:
+        """Rename the connection.
+
+        Args:
+            name: New name for the connection
+
+        Returns:
+            Updated CloudConnection object with refreshed info
+        """
+        updated_response = api_util.patch_connection(
+            connection_id=self.connection_id,
+            api_root=self.workspace.api_root,
+            client_id=self.workspace.client_id,
+            client_secret=self.workspace.client_secret,
+            name=name,
+        )
+        self._connection_info = updated_response
+        return self
+
+    def update_config(
+        self,
+        *,
+        configurations: dict | None = None,
+        prefix: str | None = None,
+    ) -> CloudConnection:
```

Comment:
On researching further, I see during creation we only offer a `selected_streams` option. We should either remove this `update_config` method entirely or replace with `set_table_prefix()` and `set_selected_streams()`.